modules/ROOT/pages/hot-deployment.adoc

```diff
@@ -0,0 +1,27 @@
+= Application Hot Deployment
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: runtime, deploy, hot deployment, application, applications
+
+You can now modify your configuration files and custom classes and have them reloaded without having to restart Mule.
+
+== Outline of a Deployment
+
+Here is a quick summary for deploying an app 'foo':
+
+* Create a directory under `$MULE_HOME/apps/foo`
+* Place your packaged application (e.g. `foo.jar`) at `$MULE_HOME/apps`
+* Jar custom classes (if any), and put them under `$MULE_HOME/apps/foo/META-INF/mule-src/foo/src/main/java/`
```

Comment:
It's Jar being used as a verb here? Wouldn't it be better something like: compress to a jar

Reply:
```suggestion
* Compress to a jar custom classes (if any), and put them under `$MULE_HOME/apps/foo/META-INF/mule-src/foo/src/main/java/`
```